translations/es/README.md

```diff
@@ -145,10 +144,98 @@ Para encontrar el identificador único de una extensión:
 - Ingresa a la página web de la extensión, por ejemplo [https://marketplace.visualstudio.com/items?itemName=ms-python.black-formatter](https://marketplace.visualstudio.com/items?itemName=ms-python.black-formatter&WT.mc_id=academic-77460-alfredodeza)
 - Localiza el campo *Unique Identifier* bajo la sección **More info** en tu lado derecho.
 
+## 🤖 Utiliza Copilot para crear una lección para alumnos
+GitHub Copilot ya está disponible en GitHub Codespaces. Puedes utilizar Copilot para crear una clase para tus alumnos. Este repositorio incluye la extensión de Copilot para que puedas utilizarla inmediatamente. Asegúrate de que tu cuenta tiene acceso a Copilot. Si no tienes acceso, puedes [solicitar acceso aquí](https://github.com/login?return_to=%2Fgithub-copilot%2Fsignup).
+
+GitHub Copilot es gratis para estudiantes y profesores. [Más información](https://education.github.com/pack/offers). Sigue [estos pasos](https://techcommunity.microsoft.com/t5/educator-developer-blog/step-by-step-setting-up-github-student-and-github-copilot-as-an/ba-p/3736279?WT.mc_id=academic-0000-alfredodeza) para verificar tu afiliación como estudiante o profesor y activar Copilot de forma gratuita.
```

Comment:
please, include the spanish article in the tech community link: https://aka.ms/GitHubCopilot4Edu